services/api/Dockerfile

```diff
@@ -17,6 +17,7 @@ ENV PYTHONFAULTHANDLER=1 \
 # System deps:
 RUN apt-get update \
     && apt-get install -y build-essential unzip wget \
+    libicu-dev ffmpeg libavcodec-extra libsndfile1 llvm pkg-config \
```

Comment:
now that every service and job depends on libcommon, which in turn requires datasets with the audio and image extras, do we need to install these packages in all the docker images? Is the poetry install working as expected without the packages?

Reply:
Also: do we need all these packages?

Reply:
Yes we need those,I had errors if they were not all installed

Reply:
OK. And thus: do we need to add those lines in the other dockerfiles?

Reply:
OK